### ISSUE TITLE
.vue Components with empty lang="scss" style tag crash gulp on build 

### ISSUE THREAD

**Issue by VanceCole** [ForgeVTT/fvtt-module-vueport#2]
Should reproduce with following component
```
<template>
  <div class="example">
    test
  </div>
</template>

<script>
export default {
  data() {
    
  }
}
</script>

<style lang="scss">
</style>
```

**Comment by kakaroto** [comment]
Yeah, gulp also fails if the file is empty, and the `scoped` portion doesn't work, and if we add a `style` section, then we can't use a `beforeCreate` hook anymore in the vue component.
I blame it all on the vue component module for gulp, which is old, outdated and sucks (but just does the job), and is one of the reasons I want to eventually investigate the use of webpack. Gulp does the job to get us started, but it has some quirks that become more apparent as we require more complex things.
